packages/@o3r/workspace/package.json

```
@@ -124,6 +124,7 @@
   },
   "generatorDependencies": {
     "@angular/material": "~18.2.0",
+    "@angular-builders/jest": "~18.0.0",
```

```suggestion
    "@angular-builders/jest": "~18.0.0",
    "@angular/material": "~18.2.0",
```

to keep alphabetic order